config/config.go

```diff
@@ -69,6 +69,10 @@ func NewDefaultConfig() *AgentConfig {
 			Pretty: false,
 			Level:  "info",
 		},
+		Cluster: ClusterConfig{
+			Port: 7896,
+			Host: "127.0.0.1",
```

Comment:
Forming cluster with localhost will not work in a distributed network isolated system like k8s pod